controllers/ocsinitialization/ocsinitialization_controller_test.go

```diff
@@ -229,7 +230,7 @@ func TestCreateSCCs(t *testing.T) {
 		ocs, request, reconciler := getTestParams(false, t)
 
 		if tc.sscCreated {
-			ocs.Status.SCCsCreated = false
+			ocs.Status.SCCsCreated = true
 			//err := reconciler.Client.Update(context.TODO(), &ocs)
 			//assert.NoErrorf(t, err, "[%s]: failed to update ocsInit status", tc.label)
```

Comment:
Can you move this commit to the end and remove the commented code in one of the other commits? This PR effectively explains why they were removed.

Reply:
Done.

Reply:
interestingly, now after this is done, the previously commented out code is not removed but uncommented again...

Reply:
Those lines need to be there. They set the precondition for the test and then we do the reconcile to check if the reconciler brings the ocsinitialization to the required state. I have updated the test to check for the precondition and state after the reconciler is run.